### DISCUSSION TITLE
GPIO input of Daheng Imaging cameras

### DISCUSSION THREAD

**Discussion by bastijnvandenboom** [bonsai-rx/bonsai#1707] [Q&A]
Hi all,
We've recently started to work with Daheng Imaging cameras (specifically, MER2_160_227U3M) since FLEA3's are discontinued. Their SDK (Galaxy 2) supports DirectShow and works easily with Bonsai's VideoCaptureDevice. However, it seems to be impossible to acquire GPIO pin status (TTLs). Has anyone a package that does include this option? 
Thanks a lot!!

**Comment by glopesdev** [comment]
@bastijnvandenboom I have never seen these cameras before, but just wondering why you didn't move to BlackFly S from FLEA3? In that case you wouldn't have had to change SDK at all and everything is supported in Bonsai.

**Comment by bastijnvandenboom** [comment]
Thanks for your quick reply! True, we have some BlackFly S's which are indeed supported in Bonsai. We got some Daheng camera's with a lens order for the BlackFly S's and it turns out their specs are better for a lower price. Was hoping there was a simple way of getting GPIO pin input!
